### ISSUE TITLE
[Suggestion] JEI Integration

### ISSUE THREAD

**Issue by vizthex123** [dennisblokland/Curio#19]
Specifically for the Infusing Table, and info panels for how to get the Crystals; as well as how to upgrade a Mystic Crystal.

Maybe update the guide book? It's very basic and annoying to navigate rn.

**Comment by dennisblokland** [comment]
Its a good idea! however i don't actively work on this project anymore. And if i work on it again i will put focus on porting it to a new version.

**Comment by vizthex123** [comment]
Yeah, I thought so....guess I'll have to find an alternative then.
